src/components/About.jsx

```diff
@@ -1,115 +1,20 @@
-      voluptatibus quidem vitae delectus officia saepe consectetur, quam nostrum
-      quod quaerat velit maiores tenetur? Officiis quibusdam soluta iste quo
-      officia explicabo non, accusantium laudantium dolore qui ut recusandae
-      incidunt necessitatibus porro et unde in aliquam provident assumenda nisi
-      nam, adipisci architecto quidem magnam! Debitis deleniti odio quis qui
-      repellendus libero consequuntur aperiam laboriosam eos, perspiciatis eius?
-      Adipisci cumque quia ipsa voluptatibus, illum molestias temporibus
-      similique ad accusamus ipsum repudiandae enim doloribus sed, eius sapiente
-      ex optio magni, et porro? Quos at nobis, odio cum facere sunt aut nesciunt
-      quam aliquam, placeat sit recusandae earum ipsum error excepturi id,
-      laudantium nihil praesentium ullam quis dignissimos. Deleniti obcaecati
-      sequi esse qui quam, eaque tenetur similique non dolorum vel eius minus at
-      possimus quia velit unde molestias atque debitis ipsam consequuntur nemo
-      enim perspiciatis? Dolorem minima veniam reprehenderit est cumque. Quidem,
-      numquam molestiae corporis earum animi aspernatur voluptas consectetur
-      corrupti totam eligendi dolores impedit adipisci laboriosam itaque, cum
-      nobis quisquam consequatur. Corporis dolor natus dicta aut doloremque
-      nulla tempore accusamus voluptatem esse harum quasi, quibusdam repellat
-      labore quis odit optio sed laudantium cumque eveniet quia fugiat ducimus.
-      Asperiores ex nesciunt velit. Similique dicta vitae enim ipsa quasi iure
-      sint earum ut repudiandae ex, doloribus assumenda, perspiciatis suscipit
-      laboriosam! Ut aspernatur, sint deleniti facere, alias architecto possimus
-      quod necessitatibus provident maiores repellat dignissimos numquam
-      obcaecati totam molestias quae, voluptatem qui. Voluptate inventore animi
-      eaque sint. Suscipit atque ipsam et distinctio tempore ullam inventore
-      iste dignissimos? Aspernatur, minus.
+    <div className="about-text">
+      <div className="paragraph-container">
+        <h1>Welcome to Anatomy 3D</h1>
+        <img src="/src/Assets/logo.jpeg" width={"60%"}/>
+        <p>You'll embark on an exciting journey through the inner workings of the human body!</p>
+        <p>In this game, designed for curious minds like yours, you'll have the opportunity to explore and learn about various
+          internal parts of the body, specifically organs and bones.</p>
+        <p>Your challenge? It's all about speed and accuracy! The quicker you can correctly place these body parts, the faster you'll progress
+          to the next level of the game. But don't worry, it's all in good fun!</p>
+        <p>So get ready to dive into the fascinating world of anatomy, challenge yourself, and discover the wonders of the human body in a whole new way.
+          Let's begin our adventure together in Anatomy 3D!</p>
+      </div>
```

Comment:
Add margin to the bottom of text, and center everything otherwise looks good! Also clever summary :wink: